### ISSUE TITLE
Bad config encountered during initialization

### ISSUE THREAD

**Issue by aleksejlopasov** [GoogleCloudDataproc/jupyterhub-dataprocspawner#22]
Now for some reasons the service inside the container cannot be started due to the following error:
`...
[C 2020-08-12 10:50:53.531 JupyterHub application:90] Bad config encountered during initialization:
[C 2020-08-12 10:50:53.532 JupyterHub application:91] The 'spawner_class' trait of <jupyterhub.app.JupyterHub object at 0x7fb719523748> instance must be a type, but 'dataprocspawner.DataprocSpawner' could not be imported
...`
The code changes now can't be tested, please fix.

**Comment by JerryLeiDing** [comment]
I haven't been able to reproduce this.

How are you starting the container? Is this on an AI Platforms Notebook instance or is this running locally?
Also I see you referenced this in your PR: does this happen when testing the PR only or also at HEAD?

**Comment by aleksejlopasov** [comment]
I used the instructions in README:
```
git clone git@github.com:GoogleCloudDataproc/jupyterhub-dataprocspawner.git
cd jupyterhub-dataprocspawner/
pip3 install .
PROJECT_ID="<my_project_id>"
VM_NAME="vm-spawner"
bash deploy_gce_example.sh ${PROJECT_ID} ${VM_NAME}
```

After the instance has been created, I login with SSH and inspect running containers. Screens attached:
<img width="1440" alt="Screen Shot 2020-08-17 at 3 54 49 PM" src="https://user-images.githubusercontent.com/20720498/90399005-efc1ca00-e0a2-11ea-9753-bfaf2f2a9718.png">
Here is partial screen of `docker logs <container_id>` for dataprocspawner container:
<img width="1440" alt="Screen Shot 2020-08-17 at 3 55 41 PM" src="https://user-images.githubusercontent.com/20720498/90399024-f9e3c880-e0a2-11ea-9273-d5f88f034720.png">

**Comment by m-mayran** [comment]
Thanks for the additional details.
I have a fix but it will take a few days for us to deploy it as we need to ensure that it does not break other dependencies.

**Comment by m-mayran** [comment]
Should be fixed by #23
